### ISSUE TITLE
Team - Friends (Submission)

### ISSUE THREAD

**Issue by mzminhaz5683** [Mahedi-61/MIU_Fall_2019#15]
5th Submission:
Score : 0.80510

**Comment by Mahedi-61** [comment]
Good. 
You don't need to open a separate issue for each new submission. 
Just update the previous one.

**Comment by mzminhaz5683** [comment]
ok sir.... but I need to open a issue now...

![Screenshot from 2019-11-20 19-17-39](https://user-images.githubusercontent.com/52498840/69319818-254b9b00-0c6a-11ea-84da-0e87b0cb9d7c.png)

I want to control the learning rate... i want to reduce learning rate after every 20 epochs.. how can I do that?

**Comment by Mahedi-61** [comment]
There are many ways you can reduce learning rates.
Say, you want to reduce learning rates after 20 epochs. (That's called **step reduce**)

In Keras you can accomplish writing you own code or simply calling its callback function `LearningRateScheduler`

Let's use `LearningRateScheduler`

Frist import necessary libraries. 
```
from keras import backend as K
from keras.callbacks import LearningRateScheduler
```

define a function to do your job
```
def lr_scheduler(epoch):
    if (epoch == 20):
        K.set_value(model.optimizer.lr, config.lr_1)
    elif (epoch == 40):
        K.set_value(model.optimizer.lr, config.lr_2)

    print("learning rate: ", K.get_value(model.optimizer.lr))
    return K.get_value(model.optimizer.lr)
```

Now, call this function 
`change_lr = LearningRateScheduler(lr_scheduler)`

Now, in your `model.fit()` you have add an additional callback, therefore
`callbacks = [early_stopping, model_cp, change_lr]`

That's do your job. Oh, Don't forget to add
`lr_1 , lr_2` values in your `config.py`

**Comment by mzminhaz5683** [comment]
ok sir.. thanks.. but I have a confusion ... what I have to assign in lr_1, lr_2 in config.py.. I mean what will be the values of lr_1, lr_2

**Comment by Mahedi-61** [comment]
`lr_1` the learning rate value you want to assign after 20 epochs. and so,
`lr_2` value after 40 epochs

**Comment by mzminhaz5683** [comment]
ok sir.... thank u so much
